Formula/sdl2_sound.rb

```diff
@@ -5,7 +5,10 @@ class Sdl2Sound < Formula
   url "https://github.com/icculus/SDL_sound/archive/06c8946983ca9b9ed084648f417f60f21c0697f1.tar.gz"
   version "2.0.1"
   sha256 "41f4d779192dea82086c8da8b8cbd47ba99b52cd45fdf39c96b63f75f51293e1"
-  license "Zlib"
+  license all_of: [
+    "Zlib",
+    any_of: ["Artistic-1.0-Perl", "LGPL-2.1-only"], # timidity
```

Comment:
Not sure if it is LGPL-2.1-only or LGPL-2.1-or-later. Current tarball actually only has copy of Perl license, but upstream has added some text on dual-license: https://github.com/icculus/SDL_sound/commit/ebd10714158abc77dea6118cea76a071b2622c2c

Reply:
Usually `only` is the safer option but in this case I think it's `-or-later`. It doesn't specify any particular version - just says "LGPL". The original source was also "GPL-2.0-or-later", which is indicitive that LGPL probably is the same.

Seems Ubuntu agrees with that interpretation: https://changelogs.ubuntu.com/changelogs/pool/universe/libs/libsdl2-mixer/libsdl2-mixer_2.0.4+dfsg1-2build1/copyright.